src/components/CustomChannelComponent.tsx

```diff
@@ -51,7 +51,7 @@ const Root = styled.div<RootStyleProps>`
     background-color: ${({ theme }) => theme.bgColor.chatBottom};
   }
 
-  .sendbird-message-input-wrapper__message-input {
+  && .sendbird-message-input-wrapper__message-input {
```

Comment:
can I get reference to what this does?

Reply:
Yeah sure https://sendbird.slack.com/archives/C050Z6DA70A/p1711433846470019